apps/backend/.env.example

```diff
@@ -24,3 +24,5 @@ JWT_ALGORITHM="HS256"
 # ENCRYPTION
 #
 ENCRYPTION_SALT_ROUNDS=10
+ENCRYPTION_ALGORITHM=aes-128-gcm
+ENCRYPTION_SECRET=123456789abcdefg
```

Comment:
In case we have `aes-128-gcm` algorithm, key should consist of `128` bits, which is `16` bytes, so key length should be 16. If I use `secret_key`, crypto package throws error `Invalid key length.

I will use `secret_key_12345` then for now. Also it looks like I should leave a comment about key length